lib/series_relationship.rb

```diff
@@ -80,7 +80,7 @@ def new_dependents
     results = []
     DataSource.where('description RLIKE ?', "[[:<:]]#{self.name.gsub('%', "\\%")}").each do |ds|
       s = Series.find_by id: ds.series_id
-      results.push s.name
+      results.push(s.name) if s
```

Comment:
We had data_sources with no series? Seems like something to clean up.

Reply:
Well, we had one. I deleted it.

Reply:
if the active record relationship was set correctly/working, seems like this could become
```
DataSource.where('description RLIKE ?', "[[:<:]]#{self.name.gsub('%', "\\%")}").joins(:series).pluck(:name).uniq
```